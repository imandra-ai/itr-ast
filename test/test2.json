{"msgs":[],"instructions":[{"Action":{"name":"Action:forceLimitOrder","fields":[]}},{"Set":{"prop":"event_0","value":{"Rec_record":{"Record":{"name":"Action:forceLimitOrder","elements":[]}}}}},{"Action":{"name":"Action:forceMinQtyLessThanOrderQty","fields":[]}},{"Set":{"prop":"event_1","value":{"Rec_record":{"Record":{"name":"Action:forceMinQtyLessThanOrderQty","elements":[]}}}}}],"job_uuid":"local_uuid","version":1,"script_name":"Limit_Order_Restrict_Min_Qty__Fill__Replace.36 (Limit_Order_Restrict_Min_Qty__Fill__Replace) 2.json","pretty_printed":"\nset event_0 = {\n  \n}\n\nset event_1 = {\n  \n}"}
