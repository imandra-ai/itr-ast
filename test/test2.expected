
set event_0 = {
  
}

set event_1 = {
  
}
